aiida/backends/general/migrations/utils.py

```diff
@@ -147,3 +146,100 @@ def recursive_datetime_to_isoformat(value):
 def dumps_json(dictionary):
     """Transforms all datetime object into isoformat and then returns the JSON."""
     return json.dumps(recursive_datetime_to_isoformat(dictionary))
+
+
+def get_duplicate_uuids(table):
```

Comment:
These utilities have moved - probably they should be importable through the old location for a transition period. Not really sure if there's anyone outside of `aiida-core` using these, though.

Reply:
These aren't part of the public API, so officially we don't guarantee stability so don't think we need to do anything here